cmd/mirror.go

```diff
@@ -190,10 +191,38 @@ func delComp(repo, id, version string) error {
 	return nil
 }
 
-// the `repo publish` sub command
+// the `mirror set` sub command
+func newMirrorSetCmd() *cobra.Command {
+	root := ""
+	force := false
+	cmd := &cobra.Command{
+		Use:   "set <mirror-addr>",
+		Short: "set mirror address",
+		Long:  "set mirror address, will replace the root certificate",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 1 {
+				return cmd.Help()
+			}
+
+			addr := args[0]
+			profile := localdata.InitProfile()
```

Comment:
I don't think we need to re-initialize a new `Profile`. There is an another way to retrieve the `Profile` via `environment.GlobalEnv().Profile()`.